Course/Course/en.lproj/Localizable.strings

```diff
@@ -34,6 +34,7 @@
 "ALERT.DELETE_ALL_VIDEOS" = "Are you sure you want to delete all video(s) for";
 "ALERT.DELETE_VIDEOS" = "Are you sure you want to delete video(s) for";
 "ALERT.STOP_DOWNLOADING" = "Turning off the switch will stop downloading and delete all downloaded videos for";
+"ALERT.WARNING" = "Warning";
```

Comment:
Please also add same string for `uk.lproj/Localizable.strings` file.

Reply:
donet, please check again